itou/www/job_seekers_views/views.py

```diff
@@ -696,6 +705,14 @@ def post(self, request, *args, **kwargs):
 
         return self.render_to_response(context)
 
+    # TODO(ewen): remove this method overloading when migration is over
+    def get_back_url(self):
+        view_name = self.previous_hire_url if self.hire_process else self.previous_apply_url
+        return reverse(
+            view_name,
+            kwargs={"session_uuid": self.job_seeker_session.name},  # removed the company_pk param
```

Comment:
```suggestion
        kwargs = {"session_uuid": self.job_seeker_session.name}
        if not self.job_seeker_session.get("apply", {}).get("company_pk"):
            kwargs["company_pk"] = self.company.pk
        return reverse(view_name, kwargs=kwargs)
```
(et on pourra enlever cela quand toutes les sessions auront bien ce `apply.company_pk` (dans 1 semaine).